lib/factory-helper/string.rb

```diff
@@ -1,28 +1,30 @@
 module FactoryHelper
   class String
-    def self.random(length= 32)
-      if length.class.included_modules.include? Enumerable
-        utf8string length.to_enum.to_a.sample
-      else
-        utf8string length
+    class << self
+      def random(length= 32)
+        if length.class.included_modules.include? Enumerable
+          utf8string length.to_enum.to_a.sample(:random => FactoryHelper::Config.random)
+        else
+          utf8string length
+        end
       end
-    end
 
-  protected
+      private
```

Comment:
I think the use-case here perfectly fits a protected method, but I agree it's better not to be listed in `object.methods`.